jmix-gridexport/gridexport-flowui/src/main/java/io/jmix/gridexportflowui/exporter/AbstractAllRecordsExporter.java

```diff
@@ -90,9 +136,108 @@ public LoadContext generateLoadContext(DataUnit dataUnit) {
         PropertyCondition lastPkCondition = PropertyCondition.createWithParameterName(primaryKeyName,
                 PropertyCondition.Operation.GREATER, LAST_LOADED_PK_CONDITION_PARAMETER_NAME);
         wrappingCondition.add(lastPkCondition);
-        loadContext.getQuery().setCondition(wrappingCondition);
+        query.setCondition(wrappingCondition);
+        query.setFirstResult(0);
 
         return loadContext;
     }
+
+    /**
+     * Method loads all entity instances associated with the given {@code dataUnit} and calls the
```

Comment:
1. No need in word "Method". 
2. *responsibility of the function* - what function?